calico_containers/pycalico/datastore.py

```diff
@@ -82,8 +82,12 @@
 DEFAULT_LOG_SEVERITY_SCREEN = "info"
 DEFAULT_LOG_FILE_PATH = "none"
 
-# Default global IP in IP.
-DEFAULT_IP_IN_IP = "true"
+# Default global IP in IP.  By default this is turned off and turned on as
+# soon as it is required by an IP Pool.
+DEFAULT_IP_IN_IP = "false"
+
+# IP in IP enabled value.
+IP_IN_IP_ENABLED = "true"
```

Comment:
Maybe have `IP_IN_IP_DISABLED` also and set `DEFAULT_IP_IN_IP = IP_IN_IP_DISABLED`